subsystems/visionsubsystem.py

```diff
@@ -187,29 +187,33 @@ def update(self) -> None:
             self.targetDistance = None
 
         if self.targetAngle is not None:
-            SmartDashboard.putNumber(
-                constants.kBallAngleRelativeToRobotKeys.valueKey,
-                convenientmath.normalizeRotation(self.targetAngle).radians(),
-            )
-            SmartDashboard.putBoolean(
-                constants.kBallAngleRelativeToRobotKeys.validKey, True
-            )
+            # SmartDashboard.putNumber(
+            #     constants.kBallAngleRelativeToRobotKeys.valueKey,
+            #     convenientmath.normalizeRotation(self.targetAngle).radians(),
+            # )
+            # SmartDashboard.putBoolean(
+            #     constants.kBallAngleRelativeToRobotKeys.validKey, True
+            # )
+            pass
         else:
-            SmartDashboard.putBoolean(
-                constants.kBallAngleRelativeToRobotKeys.validKey, False
-            )
+            # SmartDashboard.putBoolean(
+            #     constants.kBallAngleRelativeToRobotKeys.validKey, False
+            # )
+            pass
 
         if self.targetDistance is not None:
-            SmartDashboard.putNumber(
-                constants.kBallDistanceRelativeToRobotKeys.valueKey, self.targetDistance
-            )
-            SmartDashboard.putBoolean(
-                constants.kBallDistanceRelativeToRobotKeys.validKey, True
-            )
+            # SmartDashboard.putNumber(
+            #     constants.kBallDistanceRelativeToRobotKeys.valueKey, self.targetDistance
+            # )
+            # SmartDashboard.putBoolean(
+            #     constants.kBallDistanceRelativeToRobotKeys.validKey, True
+            # )
+            pass
         else:
-            SmartDashboard.putBoolean(
-                constants.kBallDistanceRelativeToRobotKeys.validKey, False
-            )
+            # SmartDashboard.putBoolean(
+            #     constants.kBallDistanceRelativeToRobotKeys.validKey, False
+            # )
```

Comment:
comments go bye bye :)

Reply:
these were only commented because they weren't necessary for this competition, they're still needed for ball tracking if we ever intend to use it

Reply:
goal of #83